src/python/WMComponent/JobSubmitter/JobSubmitAPI.py

```diff
@@ -0,0 +1,24 @@
+from __future__ import print_function, division
+import logging
+from WMCore.DAOFactory import DAOFactory
+from WMCore.Services.PyCondor.PyCondorAPI import getScheddParamValue
+
+
+def numOfJobsSubmitUptoScheddLimit(dbi, logger=logging, condorFraction=1):
+    """
+    check executiing jobs and campare with condor limit.
+    return the differece executing jobs - (condor limit * condorFraction)
+    """
+    action = DAOFactory(package='WMBS',
+                        logger=logger,
+                        dbinterface=dbi)(classname="Jobs.GetCountByState")
+    executingJobs = int(action.execute("executing"))
+
+    maxScheddJobs = int(getScheddParamValue("MAX_JOBS_PER_OWNER"))
```

Comment:
I wonder whether it really returns None in case it fails to fetch that parameter...
Anyways, my point is that this casting to int will crash if the returned value is None (my mistake btw :)).
How about we cast after the if below or in the L23?